src/main/java/com/github/timm/cucumber/generate/CucumberITGenerator.java

```diff
@@ -188,9 +186,14 @@ private String createFormatStrings() {
 
         for (int i = 0; i < formatStrs.length; i++) {
             final String formatStr = formatStrs[i].trim();
-            sb.append(String.format("\"%s:%s/%s.%s\"", formatStr,
-                            config.getCucumberOutputDir().replace('\\', '/'), fileCounter,
-                            formatStr));
+
+            if ("pretty".equalsIgnoreCase(formatStr)) {
+                sb.append("\"pretty\"");
```

Comment:
Yeah, I'm concerned about creating configuration parameter overload, but having a separate property for an optional console plugin makes sense.

By default, it should be empty/omitted.

The alternative would be to maintain a set of `console output` plugins that are only ever considered to output to the console (e.g. [ `pretty`, `progress` ]), but that seems unnecessarily limiting.